frontend/src/data/config/DatasetMetadata.ts

```diff
@@ -50,55 +50,55 @@ export const datasetMetadataList: DatasetMetadata[] = [
   {
     id: "cdc_hiv_data-race_and_ethnicity_county_time_series",
     name: "HIV Diagnoses by race/ethnicity and county",
```

Comment:
These should all indicate diagnoses, deaths and PREP. Also, let's lowercase `Diagnoses` to keep things sentence cased